quarkus-app/src/main/resources/META-INF/resources/css/homedir.css

```
@@ -6422,3 +6422,39 @@ footer {
+  pointer-events: none;
+  opacity: 0.6;
+  position: relative;
+}
+
+button.is-loading::after {
+  content: '';
+  display: inline-block;
+  width: 0.75rem;
+  height: 0.75rem;
+  margin-left: 0.25rem;
+  border: 2px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)